README.md

```diff
@@ -5,15 +5,16 @@ The Flink CDC Connectors integrates Debezium as the engine to capture data chang
 
 This README is meant as a brief walkthrough on the core features with Flink CDC Connectors. For a fully detailed documentation, please see [Documentation](https://ververica.github.io/flink-cdc-connectors/master/).
 
-## Supported (Tested) Connectors
-
-| Database | Version |
-| --- | --- |
-| MySQL | Database: 5.7, 8.0.x <br/>JDBC Driver: 8.0.16 |
-| PostgreSQL | Database: 9.6, 10, 11, 12 <br/>JDBC Driver: 42.2.12|
-| MongoDB | Database: 3.6, 4.x, 5.0 <br/>MongoDB Driver: 4.3.1|
-| Oracle | Database: 11, 12, 19 <br/>Oracle Driver: 19.3.0.0|
-| Sqlserver | Database: 2017, 2019 <br/>JDBC Driver: 7.2.2.jre8|
+## Supported (Tested) Databases
+
+| Protocol   | Connector                                                 | Database                                                                                                                                                                                                                                                                                                                                                                                               | Driver                  |
```

Comment:
We shouldn't use `protocol` word here after think twice. We always say one database vendor is compatible with MySQL but don’t say it follows MySQL protocol, and other databases are not protocols.